hs_swat_modelinstance/forms.py

```diff
@@ -0,0 +1,341 @@
+__author__ = 'Mohamed Morsy'
+from django.forms import ModelForm
+from django import forms
+from crispy_forms.layout import *
+from crispy_forms.bootstrap import *
+from models import *
+from hs_core.forms import BaseFormHelper
+
+#SWATmodelParameters element forms
+class ModelOutputFormHelper(BaseFormHelper):
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None,  *args, **kwargs):
+
+        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+        field_width = 'form-control input-sm'
```

Comment:
this is repeated over and over in lots of classes. Maybe you want a Field subclass?